spacy_streamlit/visualizer.py

```diff
@@ -198,10 +235,14 @@ def visualize_ner(
             default=list(labels),
             key=f"{key}_ner_label_select",
         )
+        if not displacy_options:
+            displacy_options = dict()
+
```

Comment:
Mostly just a stylistic nitpick!

```suggestion
            displacy_options = {}
```